components/main-with-sidebar/main-with-sidebar.html

```diff
@@ -0,0 +1,33 @@
+<div class="mzp-l-content mzp-has-sidebar mzp-l-sidebar-left">
+  <main class="mzp-l-main">
+    <p>Main content.</p>
+    <p>The sidebar comes <strong>after</strong> the main section in the source order.</p>
+  </main>
+
+  <aside class="mzp-l-sidebar">
+    <p>Sidebar content.</p>
+  </aside>
+</div>
+
+<div class="mzp-l-content mzp-has-sidebar mzp-l-sidebar-right">
+  <main class="mzp-l-main">
+    <p>Main content.</p>
+    <p>The sidebar comes <strong>after</strong> the main section in the source order.</p>
```

Comment:
**Suggestion:** I know you moved the last point from this file's notes to be reflected in this view (good idea btw, I got a little lost fiddling with sidebar when I used it for something), however I think it would also be good to keep it in the notes section, so it can stay as a one-source-of-truth for this layout?

Reply:
It's one line and this isn't a true variant, so I'm easily swayed here, but my general feeling is it's better to break up documentation where possible and not adhere to a comprehensive one-source-of-truth on the default notes view. It's a feeling based on personal preference, as I know my eyes glaze over a bit when docs go too long.